nvm.sh

```diff
@@ -3543,7 +3543,7 @@ nvm() {
       command rm -f "${NVM_DIR}/v*" "$(nvm_version_dir)" 2>/dev/null
       nvm_echo 'nvm cache cleared.'
     ;;
-    "version")
+    "version" | "v")
```

Comment:
`nvm version` and `nvm --version` are two very different commands; this is adding the alias for `nvm version`.

Instead, this should be added on line 3584.

Reply:
Thanks, I changed it.